coalib/bears/meta.py

```diff
@@ -27,3 +27,5 @@ def __init__(cls, clsname, bases, clsattrs, *varargs, aspects=None):
             cls.aspects = defaultdict(
                 lambda: aspectlist([]),
                 ((k, aspectlist(v)) for (k, v) in dict(aspects).items()))
+            for aspect_list in cls.aspects.values():
+                aspect_list.bear = cls
```

Comment:
Like mentioned in `aspectlist.__contains__`, better add `bear` to `aspectlist.__init__` and change `defaultdict` parameters to `lambda: aspectlist(cls, [])` and `((k, aspectlist(cls, v) for ...)`